### DISCUSSION TITLE
CTkFrame fg_color not totally in dark

### DISCUSSION THREAD

**Discussion by adrienpolytechsorbonne** [TomSchimansky/CustomTkinter#966] [Q&A] [answered]
Hello everybody,

I'm trying to use CustomTkinter for a neural-network project but I have issues when using CTkFrame method 

A portion of my code :
`def __init__(self,window,size,title,logo,imglight,imgdark):
        self.window = window
        self.window.geometry(size)
        self.width = 2400
        self.height = 1200
        self.title = title
        self.image_logo = logo
        self.imagelight = imglight
        self.imagedark = imgdark

        self.window.iconbitmap(self.image_logo)
        self.create_frame_accueil(self.width,self.height/2,self.imagelight,self.imagedark)
        self.frame2=customtkinter.CTkFrame(self.window,width=self.width,height=self.height,fg_color="#000000")
        self.frame2.pack(padx=0,pady=0)
        self.button1=customtkinter.CTkButton(self.frame2,text="Testez-moi",command=self.select_cam,text_color="#000000",bg_color="#0048ab",font=customtkinter.CTkFont(family="Torus Notched",size=18, weight="bold"))
        self.button1.grid(row=0,column=1,padx=0,pady=50)

        self.frame3=customtkinter.CTkFrame(self.window,width=self.width,height=self.height,fg_color="#000000")
        self.frame3.pack(padx=0,pady=0)
        self.text_box1 = customtkinter.CTkTextbox(self.frame3,font=customtkinter.CTkFont(family="Torus Notched",size=16, weight="bold"))
        self.text_box1.grid(row=0,column=0,padx=0,pady=0)
        self.text_box1.insert("0.0", "Cliquez sur le bouton de droite pour avoir un aperçu de la détection d'objets sous YOLOv5") 
        self.window.mainloop()

    def create_frame_accueil(self,frame_width,frame_height,imglight,imgdark):
        self.frame1=customtkinter.CTkFrame(master=self.window,width=frame_width,height=frame_height*1.2,fg_color="#000000")
        self.frame1.grid(row=0,column=0,padx=20,pady=20)
        self.frame1.pack()
        image = customtkinter.CTkImage(light_image=PIL.Image.open(imglight),dark_image=PIL.Image.open(imgdark),size=(618,179))
        self.create_label(self.frame1,"Bienvenue dans l'interface YOLO de Capgemini",image)

    def create_label(self,frame,text,img):
        label=customtkinter.CTkLabel(master=frame,width=self.width,height=self.height/4,text=text,text_color="#FFFFFF",corner_radius=20,bg_color="#000000",image=img,compound="bottom",font=customtkinter.CTkFont(family="Torus Notched",size=30, weight="bold"))
        label.pack()

    def select_cam(self,video_source=0):
        self.video_source = video_source
        self.vid = MyVideoCapture(self.video_source)
        self.canvas = tk.Canvas(self.frame1,width=self.vid.width,height=self.vid.height,bg="#000000")
        self.delay = 15
        self.button1.configure(state="disabled")
        self.update()
    
    def update(self):
        ret,frame=self.vid.get_frame()
        if ret:
            self.photo = PIL.ImageTk.PhotoImage(image = PIL.Image.fromarray(frame))
            self.canvas.create_image(0, 0, image = self.photo, anchor = tk.NW)
            self.canvas.pack()
        self.window.after(self.delay,self.update)`

and the result in a picture :

![video_cap](https://user-images.githubusercontent.com/92585460/210015789-0dbd3344-dce1-47fb-a47d-00c618453579.png)

Is their an attribut available so as to widen the width of the button's background underlined in dark ?
Thanks in advance

**Comment by Akascape** [comment, marked as answer]
@adrienpolytechsorbonne Try to add this: `self.button1.grid(row=0,column=1,padx=0,pady=50, sticky="news")`
